src/components/NavBar/Navbar.tsx

```diff
@@ -1,28 +1,106 @@
 import Web3Status from 'components/Web3Status'
 import { useWindowSize } from 'hooks/useWindowSize'
+import { ReactNode } from 'react'
+import { NavLink, NavLinkProps, useLocation } from 'react-router-dom'
 
+import { Box } from '../../nft/components/Box'
+import { Row } from '../../nft/components/Flex'
+import { UniIcon, UniIconMobile } from '../../nft/components/icons'
 import { breakpoints } from '../../nft/css/sprinkles.css'
+import { MenuDropdown } from './MenuDropdown'
+import { MobileSideBar } from './MobileSidebar'
 import * as styles from './Navbar.css'
 
+interface NavLinkItemProps {
+  href: string
+  id?: NavLinkProps['id']
+  isActive?: boolean
+  children: ReactNode
+}
+
+const NavLinkItem = ({ href, id, isActive, children }: NavLinkItemProps) => {
+  return (
+    <NavLink
```

Comment:
This seems like a thin wrapper - could you just use `NavLink` directly?

Reply:
Is this done to consolidate the styles to use menuItem/activeMenuItem? If so,rename the component to MenuItem.